src/foundry/foundry.js/applications/formApplications/documentSheets/journalSheet.d.ts

```diff
@@ -1,109 +1,115 @@
-
-  /**
-   * Handle requests to show the referenced Journal Entry to other Users
-   * Save the form before triggering the show request, in case content has changed
-   */
-  protected _onShowPlayers(event: Event): Promise<void>;
-}
-
-declare namespace JournalSheet {
-  interface RenderOptions extends Application.RenderOptions {
-    sheetMode?: SheetMode | null;
-  }
+  class JournalSheet<
+    Options extends JournalSheet.Options = JournalSheet.Options,
+    Data extends object = JournalSheet.Data<Options>
+  > extends DocumentSheet<Options, Data> {
+    /**
+     * @param object  - The JournalEntry instance which is being edited
+     * @param options - Application options
+     */
+    constructor(object: ConcreteJournalEntry, options?: Partial<Options>);
 
-  interface Options extends DocumentSheet.Options {
     /**
-     * @defaultValue `['sheet', 'journal-sheet']`
+     * The current display mode of the journal. Either 'text' or 'image'.
+     * @internal
      */
-    classes: string[];
+    protected _sheetMode: JournalSheet.SheetMode | null;
+
     /**
-     * @defaultValue `720`
+     * The size of the application when it was in text mode, so we can go back
+     * to it when we switch modes.
+     * @internal
      */
-    width: number;
+    protected _textPos: Application.Position | null;
+
     /**
-     * @defaultValue `800`
+     * @override
+     * @defaultValue
+     * ```ts
+     * foundry.utils.mergeObject(super.defaultOptions, {
+     *   classes: ["sheet", "journal-sheet"],
+     *   width: 720,
+     *   height: 800,
+     *   resizable: true,
+     *   closeOnSubmit: false,
+     *   submitOnClose: true,
+     *   viewPermission: CONST.ENTITY_PERMISSIONS.NONE
+     * })
+     * ```
      */
-    height: number;
+    static get defaultOptions(): JournalSheet.Options;
+
+    /** @override */
+    get template(): string;
+
+    /** @override */
+    get title(): string;
+
     /**
-     * @defaultValue `true`
+     * Guess the default view mode for the sheet based on the player's permissions to the Entry
+     * @internal
      */
-    resizable: boolean;
+    protected _inferDefaultMode(): JournalSheet.SheetMode | null;
+
+    /**@override */
+    protected _render(force?: boolean, options?: Application.RenderOptions<Options>): Promise<void>;
+
     /**
-     * @defaultValue `false`
+     * @override
+     * @internal
      */
-    closeOnSubmit: boolean;
+    protected _getHeaderButtons(): Application.HeaderButton[];
+
+    /** @override */
+    getData(options?: Partial<Options>): Promise<Data> | Data;
+
+    /** @override */
+    protected _updateObject(
+      event: Event,
+      formData: JournalSheet.FormData
+    ): ReturnType<DocumentSheet<Options, Data>['_updateObject']>;
+
     /**
-     * @defaultValue `true`
+     * Handle requests to switch the rendered mode of the Journal Entry sheet
+     * Save the form before triggering the show request, in case content has changed
+     * @internal
```

Comment:
The documentation for the parameters is missing

Reply:
done